app/containers/message/utils.js

```diff
@@ -100,6 +102,10 @@ export const getInfoMessage = ({
 		return I18n.t('Room_changed_avatar', { userBy: username });
 	} else if (type === 'message_snippeted') {
 		return I18n.t('Created_snippet');
+	} else if (type === 'room_e2e_disabled') {
+		return `This room's encryption has been disabled by ${ username }`;
```

Comment:
You should add it to `en.js` and wrap it with `I18n.t()` as all the other messages.